markdown-cheat-sheet.md

```diff
@@ -11,16 +11,18 @@ These are the elements outlined in John Gruber’s original design document. All
 ### Heading
```

Comment:
This entire file needs reworking. It's nor GitHub flavoured markdown, so doesn't render everything properly. In addition, the "code" is not shown - just the rendered result. I suggest putting this in a separate issue and PR, or we'll get nowhere!

Reply:
Agreed

Reply:
Issue raised